src/components/MobileNavigation/MobileNavigation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import "./MobileNavigation.scss"
+
+function MobileNavigation() {
+    return (
+        <div className='MobileNavigation'>
+            <div className='MobileNavigation-wrapper'>
+                <a href='#' className='MobileNavigation-wrapper__link1'>Find Candidates</a>
+                <a href='#' className='MobileNavigation-wrapper__link2'>Find Jobs</a>
```

Comment:
All these classNames should change aswell e.g. The classname should be `MobileNavigation__wrapper__link1`

Reply:
@jasper-prgh Elements of elements ((block__elem1__elem2) do not exist in the BEM methodology (https://en.bem.info/methodology/faq/#why-not-create-elements-of-elements-block__elem1__elem2)
Or do I read it incorrectly?

Reply:
@tinkabel85 You did read that correctly. However that'd be way too nitpicky imho. In practice, `block__elem1__elem2` is not seen as a bad practice, hovewer, `elem1__elem2` would technically be cleaner

Reply:
The reason it does not exist in the BEM methodology is to keep classNames somewhat short.
I'd rather have a long name than having to figure out, what it does, though

Reply:
fair enough!

Reply:
There is a "Contact Us" link missing

Reply:
In figma there is no such link I think

Reply:
I believe there is, it just is not very visible
![Screenshot (220)](https://user-images.githubusercontent.com/121715056/220854027-cdcdc703-4752-4fb5-a41e-c854e2b64639.png)

Reply:
In the mobile view it is not shown. Where do you suggest I put it? Just above sign in button maybe?

@jasper-prgh @Aiyanna1

Reply:
I think just above the sign in button is good.